.github/actions/setup/action.yml

```diff
@@ -64,3 +64,7 @@ runs:
         if: ${{ inputs.java-version != '' }}
         shell: bash
         run: echo "::add-matcher::.github/problem-matcher.json"
+      - name: Free additional disk space
+        shell: bash
+        # Based on https://github.com/actions/runner-images/issues/2840
+        run: ./.github/bin/free-disk-space.sh
```

Comment:
should this be part of `.github/bin/cleanup-node.sh`?

Reply:
No. I've removed old cleanup-node.sh on the other commit

Reply:
Cleanup-node was applied for memsql only. This is applied for all jobs